test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -193,6 +239,8 @@ private CommandRunnerResult PackProjectOrSolution(string workingDirectory, strin
 
         internal void BuildProject(string workingDirectory, string projectName, string args)
         {
+            CreateTempGlobalJson(workingDirectory);
```

Comment:
I don't think it needs added here. 
It should be done at creation time.

Reply:
Thanks! Removed already. Only keep CreateTempGlobalJson in CreateDotnetToolProject  and CreateDotnetNewProject. Will check if manually fix for other tests afterwards.